apps/site/assets/ts/shuttles/components/__shuttles.d.ts

```diff
@@ -0,0 +1,20 @@
+export interface ShuttleDestinations {
+  0: string;
+  1: string;
+}
+
+export interface ShuttleStop {
```

Comment:
`ShuttleStop` makes it sound like these are the stops for the shuttles themselves, but they encompass all stops (rail and bus) related to a shuttle diversion. Maybe this could just be `Stop`? The shapes are just `Shape`, after all.